load/tests.py

```diff
@@ -110,10 +119,19 @@ def test_accepted_load(self):
     def test_available_load(self):
         response = self.client.get(
             '/load/api/carrier/available/', format="json")
-        self.assertEqual(len(response.data), 2)
+        self.assertEqual(len(response.data), 1)
 
-    def test_drop_load(self):
-        pass
+    def test_drop_load_invalid(self):
+        response = self.client.get('/load/api/carrier/1/drop/')
+        self.assertEqual(response.data, status.HTTP_201_CREATED)
+
+    def test_drop_load_invalid(self):
```

Comment:
Two tests with the same name, one of them is not being executed.

Reply:
Ok, thanks!